README.md

```diff
@@ -122,5 +122,6 @@ However, such problems are best solved with social norms, not with software
 licenses. Any attempt to complicate the software license of Zig would risk
 compromising the value Zig provides to users.
 
-Therefore, Zig is available under the MIT (Expat) License, and comes with a
-humble request: use it to make software better serve the needs of end users.
+Therefore, Zig is available under the MIT (Expat) License with the exception
+of macOS libc headers which are licensed under Apple Public Source License Version 2.0,
+and comes with a humble request: use it to make software better serve the needs of end users.
```

Comment:
I had to double read this part before it was clear what it meant. 

On my first reading, I incorrectly read it to mean; `macOS libc headers which are licensed under Apple Public Source License Version 2.0 and comes with a humble request ...`   
ie, I incorrectly read it as if the humble request is from Apple rather than been from Zig. 

For what it's worth, English is my 2nd language so this comment is more like a nitpick.

Reply:
Not at all, this is some great feedback, thanks! Lemme redo this bit so that it's less confusing!

Reply:
Rephrased! Lemme know if it makes more sense now :-)

Reply:
looks good. thanks.